cmd/root.go

```diff
@@ -111,6 +114,34 @@ func PreRun(cmd *cobra.Command, args []string) {
 func Run(c *cobra.Command, names []string) {
 	filter := filters.BuildFilter(names, enableLabel)
 	runOnce, _ := c.PersistentFlags().GetBool("run-once")
+	httpAPI, _ := c.PersistentFlags().GetBool("http-api")
+	
+	if httpAPI {
+		log.Println("Watchtower HTTP API started.")
+		
+		apiToken, _ := c.PersistentFlags().GetString("http-api-token")
+
+		http.HandleFunc("/v1/update", func(w http.ResponseWriter, r *http.Request){
```

Comment:
I think it would make sense to put this into a separate package, like `pkg/api`. What's your thought on that?

Reply:
I believe it's the best thing to do at the moment. If we let this for future contributions, it may be something that will never be done. I'll work on it.